manifests/repo.pp

```diff
@@ -13,7 +13,8 @@
         $location = $repo_location
         $description = 'MongoDB Custom Repository'
       } elsif $mongodb::globals::use_enterprise_repo == true {
-        $location = 'https://repo.mongodb.com/yum/redhat/$releasever/mongodb-enterprise/stable/$basearch/'
+        $mongover = split($version, '[.]')
```

Comment:
Version defaults to `undef` so perhaps we should raise an error if it is not set, as `use_enterprise_repo` requires it?